Src/Scripting/CachedArgument.cpp

```diff
@@ -6,6 +6,7 @@ CachedArgument::CachedArgument(const PGE::String& nm, Type* t) {
 }
 
 CachedArgument::Value::Value() {
+    truthy = false;
```

Comment:
Avoid funsy identifiers.

Reply:
The name of this member is incorrect. Rename it to b8.

Reply:
I have renamed this to bool8, and its associated references in NativeDefinitions/EventDefinition